src/main/target/STM32_UNIFIED/target.h

```diff
@@ -139,6 +139,42 @@
 #define TARGET_IO_PORTD 0xffff
 #define TARGET_IO_PORTE 0xffff
 #define TARGET_IO_PORTF 0xffff
+
+#elif defined(STM32G4X3)
+#define TARGET_BOARD_IDENTIFIER "SG43"
+
+#define USBD_PRODUCT_STRING     "Betaflight STM32G4x3"
```

Comment:
AFAIK, this firmware aims to work with G474 and G473 MCUs, so why we use `G4x3` instead?

Reply:
I took this from `G4X3_TARGETS` which is defined in `make/mcu/STM32G4.mk` as 

> `# G4X3_TARGETS includes G47{3,4}{RE,CE,CEU}`

- that's what you get for blind copy / paste. :grimacing: